tests/js-units/node/config.test.js

```diff
@@ -130,4 +130,17 @@ describe('Config', function () {
         expect(initialConfig.datavizOptions).to.be.instanceOf(DatavizOptionsConfig)
     })
 
+    it('SingleWMS Layer Config', function () {
+        const capabilities = JSON.parse(readFileSync('./data/single_wms_image-capabilities.json', 'utf8'));
+        expect(capabilities).to.not.be.undefined
+        expect(capabilities.Capability).to.not.be.undefined
+        const config = JSON.parse(readFileSync('./data/single_wms_image-config.json', 'utf8'));
+        expect(config).to.not.be.undefined
+
+        const initialConfig = new Config(config, capabilities);
+
+        expect(initialConfig.options).to.be.instanceOf(OptionsConfig)
+        expect(initialConfig.options.wms_single_request_for_all_layers).to.be.eq("True")
+    })
+
```

Comment:
@rldhont 

> And add expect(opt.wms_single_request_for_all_layers).to.be.eq(false) to Valid test of optons.test.js

This one is perfectly clear to me, I've added this part.

> Can you rewrite this test to add to js-units/config/options.test.js with expect(opt.wms_single_request_for_all_layers).to.be.eq(false) ?

About this one I'm not sure I understand what you mean, maybe there is a typo on `to.be.eq(false)`? I want to test **singleWMS = true** since singleWMS = false is the default. Should I remove this test from `js-units/node/config.test.js` and add it to` js-units/config/options.test.js` with `expect(opt.wms_single_request_for_all_layers).to.be.eq(true)`?

Thanks

Reply:
@mind84 Sorry, What I meant was to add a test in `js-units/config/options.test.js` with `expect(opt.wms_single_request_for_all_layers).to.be.eq(true)` based on this config.